bin/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+REGION=us-east-2
+ENVIRONMENT=${1:-dev}
+CLUSTER_NAME=$ENVIRONMENT-cluster0
+SERVICE_NAME=$ENVIRONMENT-cluster0-service
+TASK_FAMILY=$ENVIRONMENT-cluster0-task-def
+ECR_URL=377928551571.dkr.ecr.us-east-2.amazonaws.com/relayer-$ENVIRONMENT
+
+# Fetch the latest image URI from ECR
+IMAGE_URI=$(aws ecr describe-images --repository-name relayer-dev --region us-east-2 --image-ids imageTag=latest | jq -r '.imageDetails[0].imageTags[0]')
```

Comment:
when I ran this locally, the value returned was just the `latest` tag. it's quite nice to see the github commit hash in the task definition, but i'm not sure if we can rely on its ordering being before/after `latest`... maybe we could just use the `imageDigest` field then for the full URI? we can still match the image digest to the github hash tag in ECR when debugging